src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:nth-child(-n +4),
```

Keep spacing around operators consistent too
```suggestion
.stars--4 .stars__star:nth-child(-n + 4),
```

---

Same for selectors below